src/webviews/extension-side/plotView/plotViewHandler.ts

```diff
@@ -56,13 +55,22 @@ function getOutputItem(
 function convertPngToSvg(pngOutput: NotebookCellOutputItem): string {
     const imageBuffer = Buffer.from(pngOutput.data);
     const imageData = imageBuffer.toString('base64');
-    const dims = sizeOf(imageBuffer);
+
+    let height = imageBuffer.readUInt32BE(20);
```

Comment:
I'm curious why this had to change? Does a buffer in the web not support sizeOf?

Reply:
`image-size` uses `fs`